src/Web/Middleware/ExceptionHandlingMiddleware.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Contoso.FraudProtection.ApplicationCore.Exceptions;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Http;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Contoso.FraudProtection.Web.Middleware
+{
+    public static class ExceptionHandlingMiddleware
+    {
+        public static void ConfigureExceptionHandler(this IApplicationBuilder app)
+        {
+            app.UseExceptionHandler(errorApp =>
+            {
+                errorApp.Run(async context =>
+                {
+                    context.Response.StatusCode = 500;
```

Comment:
why are we hard coding it to 500, is it 500 in context of sample App?

Reply:
Yes, that's right.  Sample site encountered an unexpected error in either case.  In reality, users won't see this 500 unless they look in Fiddler or something anyways, but good question!